ang/civicase/Utils.js

```diff
@@ -623,6 +623,31 @@
     };
   });
 
+  module.service('DateHelper', DateHelper);
+
+  function DateHelper () {
+    /**
+     * To check if the date is overdue
+     *
+     * @param {String} date ISO string
+     * @return {Boolean} if the date is overdue.
+     */
+    this.isOverdue = function (date) {
+      return moment(date).isBefore(moment());
+    };
+
+    /**
+     * Formats Date in correct format (DD/MM/YYYY)
+     *
+     * @param {String} date ISO string
+     * @param {String} format ISO string
+     * @return {String} the formatted date
+     */
+    this.formatDate = function (date, format = 'DD/MM/YYYY') {
```

Comment:
`format = 'DD/MM/YYYY'` This is ES6 , which we are not using for now. Please use ES5.
```
this.formatDate = function (date, format) {
      format = format || 'DD/MM/YYYY';
      
      return moment(date).format(format);
    };
```